### PR TITLE
Add info about emoticon support

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-535

Description of Changes:
* Add short section about browser support for emoticons.

DOD:
- [x] nav.yml has been updated if applicable
- [x] file has been included where required if applicable
- [x] files removed have been deleted, not just excluded from the build if applicable

Review
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: plugins/emoticons.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ tinymce.init({
 });
 ```
 
+### Browser emoticon support
+
+The emoticon plugin inserts unicode character codes, such as `\ud83d\ude03` for smiley. How emoticons are rendered is dependent on the web browser and operating system of the user. As a result of this, some emoticons may rendered in black and white, or may not render.
```
Comment (lnewson) [inline plugins/emoticons.md:28]: "maybe rendered"?

File: plugins/emoticons.md [hunk 2]
```diff
@@ -23,6 +23,11 @@ tinymce.init({
...
+To ensure emoticons render consistently across browsers and operating systems, {{site.companyname}} recommends setting the default fonts to a emoticon-compatible web font using [`content_css`]({{site.baseurl}}/configure/content-appearance/#content_css).
```
Comment (lnewson) [inline plugins/emoticons.md:29]: I wouldn't say "setting the default font" here as that'll cause many other issues (eg most characters not rendering). Instead, maybe say "recommends adding an emoticon-compatible web font to the default font-family".

File: plugins/emoticons.md [hunk 3]
```diff
@@ -23,6 +23,11 @@ tinymce.init({
 });
 ```
 
+### Browser emoji support
+
+The emoticon plugin inserts unicode character codes, such as `\ud83d\ude03` for smiley. How emoji are rendered is dependent on the web browser and operating system of the user. As a result of this, some emoji may be rendered in black and white, or may not render.
```
Comment (metricjs) [inline plugins/emoticons.md:28]: "for smiley" reads a little odd?

Comment (EkimChau) [reply plugins/emoticons.md:28]: "for the smiley emoji." ?

**Pull request merged**